drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -263,11 +280,11 @@ def do_attempt(self, attempt, suffix=''):
 
         state = None
         zone = None
-        feedback = item['feedback']['incorrect']
+        feedback = item['feedback'].get('incorrect', '')
```

Comment:
@haikuginger Once you update the default exercise as mentioned above (i.e., once correct and incorrect feedback is set for all items), you should be able to revert this change.